modules/alookup/alookup.go

```diff
@@ -79,9 +78,8 @@ func (s *Lookup) doLookupProtocol(name string, nameServer string, dnsType uint16
 			}
 		}
 		for _, a := range miekgResult.(miekg.Result).Additional {
-			ans, ok := a.(miekg.Answer)
 			// filter only valid answers of requested type or CNAME (#163)
-			if ok {
+			if ans, ok := a.(miekg.Answer); ok {
```

Comment:
Noting that this change doesn't do anything functionally

Reply:
Thanks. Yeah, it was just consistent in that file.